### DISCUSSION TITLE
starting solidity

### DISCUSSION THREAD

**Discussion by gibhub01** [Cyfrin/foundry-full-course-cu#2369] [General]
who is just starting out lets connect

**Comment by EngrPips** [comment]
Trust many people would connect soon

**Reply by Nerdymaxx** [reply]
lets connect

**Comment by drplanche** [comment]
I started about 2 weeks ago! Love it so far

**Comment by Lukmanam** [comment]
> who is just starting out lets connect

I am here

**Comment by PhilippeOurselin** [comment]
Started 2 weeks ago, along with Javascript course at Alchemy University which is a great complement.

**Comment by EngrPips** [comment]
Interesting

**Comment by Tolanee** [comment]
I just started too ...yaay

**Reply by EngrPips** [reply]
Go crush it!

**Comment by jarvis888-glitch** [comment]
> who is just starting out lets connect

lets learn from each other

**Comment by EngrPips** [comment]
Interesting.
